pkg/shell/state.jsx

```diff
@@ -549,7 +549,8 @@ export function ShellState() {
             const menu_items = compiled.ordered("menu");
             if (menu_items.length > 0 && menu_items[0])
                 location.path = menu_items[0].path;
-            location.path = "system";
+            else
+                location.path = "system";
```

Comment:
This added line is not executed by any test.

Reply:
Yeah, well, this could only happen if we don't have *any* manifests, right? This could indeed just be dropped.

Reply:
Or only manifests without any "menu" section.